SmartDeviceLink-iOS/SmartDeviceLink/SDLProxy.m

```diff
@@ -295,6 +322,12 @@ - (void)handleRegisterAppInterfaceResponse:(SDLRPCResponse *)response {
     //Print Proxy Version To Console
     NSString *logMessage = [NSString stringWithFormat:@"Framework Version: %@", self.proxyVersion];
     [SDLDebugTool logInfo:logMessage withType:SDLDebugType_RPC toOutput:SDLDebugOutput_All toGroup:self.debugConsoleGroupName];
+    if ([SDLGlobals globals].protocolVersion >= 4) {
```

Comment:
I don't think anything is actually in 4, is it? I thought we scrapped 4 entirely and said 4 will be negotiable MTU.